pages/api/graph/compile-as.ts

```diff
@@ -113,31 +113,38 @@ const compile = (inputFile: string, libraries: { [fileName: string]: string }) =
 }
 
 const handler = async (req: NextApiRequest, res: NextApiResponse) => {
-  if (req.method !== 'POST') {
-    throw new Error('Only POST requests allowed')
-  }
+  try {
```

Comment:
I don't think this try/catch is necessary, as errors should be handled by the `handleErrors` wrapper function